lib/postgrex/notifications.ex

```diff
@@ -99,7 +103,8 @@ defmodule Postgrex.Notifications do
   def connect(_, opts) do
     case Protocol.connect([types: nil] ++ opts) do
       {:ok, protocol} ->
-        {:ok, %__MODULE__{protocol: protocol}}
+        idle_timeout = Keyword.get(opts, :idle_timeout, 5000)
+        {:ok, %__MODULE__{idle_timeout: idle_timeout, protocol: protocol}} |> include_idle_timeout
```

Comment:
Since we are always including the idle timeout, I would then include it directly in the tuples and remove the indirection. So this would be `{:ok, ..., idle_timeout}`.